bas/tcl/Utilities.tcl

```diff
@@ -18,18 +18,19 @@ proc ExecutePost {} {
 	set OutputStep [lindex [split $OutputStep #] 0]
 
 	set OpenSeesProblemTypePath [OpenSees::GetProblemTypePath]
+	set IsPython [OpenSees::IsPython]
 
 	cd "$OpenSeesProblemTypePath/exe"
 
 	# Run OpenSeesPost
 
 	if {[GiD_AccessValue get GenData Use_HDF5_binary_output_format] == 0 } {
 
-		exec {*}[auto_execok start] "OpenSeesPost.exe" "$param1" "$param2" "$OutputStep"
+		exec {*}[auto_execok start] "OpenSeesPost.exe" "$param1" "$param2" "$OutputStep" "$IsPython"

```

Comment:
@vpapanik the parameter IsPython is added in order to execute the correct lines in OpenSeesPOST.exe. This parameter is set to **ParamStr(4)**